
Due to gcc 3.4 optimizations some constants are optimized away - so
skip MMX optimizations for now ...

  - Rene Rebe <rene@rocklinux.org>

--- ffmpeg-0.4.9-pre1/libavcodec/liba52/resample.c.orig	2003-11-03 19:06:54.000000000 +0100
+++ ffmpeg-0.4.9-pre1/libavcodec/liba52/resample.c	2004-09-15 13:17:02.079238694 +0200
@@ -13,14 +13,14 @@
 
 #include "resample_c.c"
 
-#ifdef ARCH_X86
+#ifdef HAVEMMX_ARCH_X86
 #include "resample_mmx.c"
 #endif
 
 void* a52_resample_init(uint32_t mm_accel,int flags,int chans){
 void* tmp;
 
-#ifdef ARCH_X86
+#ifdef HAVEMMX_ARCH_X86
     if(mm_accel&MM_ACCEL_X86_MMX){
 	tmp=a52_resample_MMX(flags,chans);
 	if(tmp){
